src/Groups/GAPGroups.jl

```diff
@@ -496,14 +498,7 @@ end
 
 Return the array of the elements in C.
 """
-function elements(C::GroupConjClass{S, T}) where S where T<:GAPGroupElem
-   L=GAP.Globals.AsList(C.CC)
-   l = Vector{T}(undef, length(L))
-   for i in 1:length(l)
-      l[i] = group_element(C.X,L[i])
-   end
-   return l
-end
+elements(C::GroupConjClass{S, T}) where {S,T} = collect(C)
```

Comment:
```suggestion
elements(C::GroupConjClass) = collect(C)
```
In fact, at this point, we may as well deprecate this, and tell people to use `collect` instead of `elements. So:
```suggestion
@deprecate elements(C::GroupConjClass) collect(C)
```